### PR TITLE
Upgrade version to qualimap-build-11-11-19

### PR DESCRIPTION
**Pull request by jvhaarst**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [ ] This PR adds a new recipe.
* [ ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/qualimap/meta.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 package:
   name: qualimap
-  version: "2.2.2c"
+  version: "2.2.2d"
 
 source:
-  url: https://bitbucket.org/kokonech/qualimap/downloads/qualimap-build-03-12-18.tar.gz
-  sha256: 6b349650c19796ae31023095690425236c1f9b40f43e47303bf3bb26e15461ef
+  url: https://bitbucket.org/kokonech/qualimap/downloads/qualimap-build-11-11-19.tar.gz
+  sha256: 6fb219671421f3acbecbb56d1e06fe84fe0ca02dba0503d51ef3efb9df471983
```
Comment (bgruening) [inline recipes/qualimap/meta.yaml]: please reset the build-number to 0

Comment (jvhaarst) [reply recipes/qualimap/meta.yaml]: Done.

**Comment by jvhaarst** [comment]
Weird why it failed on the sha256sum test :

```
wget --quiet -O - https://bitbucket.org/kokonech/qualimap/downloads/qualimap-build-11-11-19.tar.gz | sha256sum -
6fb219671421f3acbecbb56d1e06fe84fe0ca02dba0503d51ef3efb9df471983  -
```

**Comment by jvhaarst** [comment]
Changing the checksum to md5 also didn't help, I'm a bit stumped at why it fails, as for me the checksums are the same across different machines and OS's.

I give up for now.

**Comment by dpryan79** [comment]
@bioconda-bot please update

**Comment by dpryan79** [comment]
@bioconda-bot please fetch artifacts

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**